lib/attrsets.nix

```diff
@@ -325,7 +325,7 @@ rec {
   /* Converts a store path to a fake derivation. */
   toDerivation = path:
     let
-      path' = builtins.storePath path;
+      path' = builtins.storePath (toString path);
```

Comment:
Is this still needed though? I wouldn't think so

Reply:
You're right, dropped that commit. It does everything we want (`__toString`, derivations, strings, paths).